src/terra-dev-site/contributing/ContributionGuidelines.b.contributing.mdx

```diff
@@ -0,0 +1,36 @@
+# Contribution Guidelines
+
+These guidelines apply to medium and large component contributions.
+
+## Contribution Kickoff
+
+A contribution kickoff involving UX, Terra, and the contributing team needs to happen up front. The goals of this meeting are:
+
+* Determine scope of the contribution
+* Allocate UX personnel
+* Allocate Terra engineering personnel
+* Establish general timelines for all parties
+
+## Allocated UX Support and Design
+
+UX needs to be involved from the beginning of the contribution. It is expected that UX will provide guidance on:
+
+* All themes
+* Accessibility
+* Design standards
+
+## Allocated Terra Engineering Support
+
+A certain percentage of a Terra engineer's time will need to be allocated from the Terra team's availability. This percentage will be based on the size and complexity of the contribution.
+
+## Contributor Expectations
+
```

Comment:
My thoughts:
- They will also be expected to provide a brief tech design / outline including all prop APIs, responsive and accessibly considerations. This will be attached to the issue they are working on
- expected to write unit & wdio test
- expected to work with UX to provide themes
- will need to write docs on the features
- follow Terra's coding standards, etc

Reply:
work with Terrra engineer to add needed translations